sktime/datasets/tests/test_data_io.py

```diff
@@ -1080,3 +1081,41 @@ def test_write_dataframe_to_ts_fail(tmp_path):
             path=str(tmp_path),
             problem_name="GunPoint",
         )
+
+
+def test_convert_tsf_to_dataframe():
+    """Test function for loading tsf format."""
+    data_path = MODULE + "/data/UnitTest/UnitTest_Tsf_Loader.tsf"
```

Comment:
The tests are failing in CI but pass locally. 
The issue has to do with the path to the test .tsf file. 

**local:**
![image](https://user-images.githubusercontent.com/64217214/152323150-30193769-759e-46fa-9019-afe38100a102.png)
**CI**
`FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Miniconda\\envs\\test\\lib\\site-packages\\sktime\\datasets/data/UnitTest/UnitTest_Tsf_Loader.tsf'`

Try this instead of adding the strings: 
```python
import os

data_path = os.path.join(MODULE, "/data/UnitTest/UnitTest_Tsf_Loader.tsf")
```

Reply:
Modified the data_path variable. The given suggestion was a bit modified and ensured that the test is passed locally.

Reply:
@fkiraly - have a look here as well. This is identical to the other test 🤯